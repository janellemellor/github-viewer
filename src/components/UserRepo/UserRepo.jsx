import React from 'react';
import PropTypes from 'prop-types';

const UserRepo = ({ name, html_url }) => (
  <li>
    <p>{name}</p>
    <p>{html_url}</p>
  </li>
);

UserRepo.propTypes = {
  name: PropTypes.string.isRequired,
  html_url: PropTypes.string.isRequired 
};

export default UserRepo;

