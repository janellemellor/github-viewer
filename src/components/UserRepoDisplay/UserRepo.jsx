import React from 'react';
import PropTypes from 'prop-types';

const UserRepo = ({ repoName, repoUrl }) => (
  <li>
    <p>{repoName}</p>
    <p>{repoUrl}</p>
  </li>
);

UserRepo.propTypes = {
  repoName: PropTypes.string.isRequired,
  repoUrl: PropTypes.string.isRequired 
};

export default UserRepo;

