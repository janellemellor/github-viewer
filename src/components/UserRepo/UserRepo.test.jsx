import React from 'react';
import { shallow } from 'enzyme';
import UserRepo from './UserRepo.jsx';

describe('UserRepo', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<UserRepo name='backend' html_url='https://github.com/mental-health-twitterbot/backend' />);
    expect(wrapper).toMatchSnapshot();
  });
});

