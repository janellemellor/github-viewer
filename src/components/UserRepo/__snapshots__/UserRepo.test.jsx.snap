// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserRepo matches a snapshot 1`] = `
<li>
  <p>
    backend
  </p>
  <p>
    https://github.com/mental-health-twitterbot/backend
  </p>
</li>
`;
